tests/integration/plugins/test_signer.py

```diff
@@ -39,6 +39,15 @@ async def verify_signature(records, timestamp, signature):
         await verifier.verify(serialized, signature["signature"], x5u)
 
 
+def fetch_changeset(client, bucket=None, collection=None, **kwargs):
+    bucket = bucket or client.bucket_name
+    collection = collection or client.collection_name
```

Comment:
Is `client` guaranteed to have `bucket_name` and `collection_name` as a fallback?

Reply:
We could just remove the `bucket` and `collection` arguments, since we pass a client that has those attributes set at instantiation